deploy/olm-catalog/ibm-block-csi-operator/1.7.0/bundle-1.7.0.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM scratch
+
+LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
+LABEL operators.operatorframework.io.bundle.channels.v1=stable
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=ibm-block-csi-operator
+
+COPY manifests /manifests/
+COPY metadata /metadata/
+LABEL com.redhat.openshift.versions="v4.7,v4.8"
```

Comment:
I think this ticket [includes](https://github.com/IBM/ibm-block-csi-operator/pull/141/files#diff-e4eb4de44c60d532c36a05b66c8931dbf5b307ca6a07cf2c1e53b81d549b1943) bumping the orchestration platforms

Reply:
Done

Reply:
then what about `minKubeVersion`?